docker/dotnet.dockerfile

```
@@ -1,6 +1,4 @@
-FROM microsoft/dotnet:2.1-sdk-alpine
-
-RUN apk add bash libc6-compat
+FROM microsoft/dotnet:2.1-sdk
```

I think the recommended way to do this nowadays is:
```suggestion
FROM mcr.microsoft.com/dotnet/core/sdk:2.1
```

Related: we should think about how to test on 2.1, 2.2, and 3.0 when it's released next month.